sdk/ml/azure-ai-ml/azure/ai/ml/_schema/job/parameterized_command.py

```diff
@@ -30,6 +31,11 @@ class ParameterizedCommandSchema(PathAwareSchema):
         ],
         required=True,
     )
-    environment_variables = fields.Dict(keys=fields.Str(), values=fields.Str())
+    environment_variables = UnionField(
```

Comment:
any other component type need to update?

Reply:
update environment_variables in ParameterizedParallelSchema